ratis-test/src/test/java/org/apache/ratis/grpc/TestLogAppenderWithGrpc.java

```diff
@@ -80,6 +80,12 @@ public void testPendingLimits() throws IOException, InterruptedException {
       final RaftServer.Division leader = waitForLeader(cluster);
       long initialNextIndex = RaftServerTestUtil.getNextIndex(leader);
 
+      JavaUtils.attempt(() -> {
+        // Make sure followers are up-to-date before blocking the appends in the follower
+        cluster.getFollowers().stream().mapToLong(RaftServerTestUtil::getNextIndex)
+            .forEach(nextIndex -> Assert.assertEquals(initialNextIndex, nextIndex));
+      }, 5, ONE_SECOND, "matching initial nextIndex", LOG);
```

Comment:
Let's make it 10 retries.
```java
}, 10, ONE_SECOND, "matching initial nextIndex", LOG);
```
and also 10 retries for line 103.

Reply:
I found it's better to use `client.io().watch()` here, see latest commits.